src/lib/services.js

```diff
@@ -87,6 +87,16 @@ export function getEnrollmentCopies(id) {
     .then(({ data: { enrollments: data } }) => data);
 }
 
+export function getNextEnrollment(id) {
```

Comment:
getNextEnrollment => getNextEnrollment**s** ?

Actually, several impot part requests can refer to the same fc request.